src/hal/getpinmap_catena4618.cpp

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Same comment as for 4617. I don't see why we need this.

Reply:
Yes, this legacy name should not be here, this is added up by mistake. I missed out this in the self review, I will remove it.